pdpserver/Makefile

```diff
@@ -0,0 +1,21 @@
+build:
+	go build
+
+.PHONY: clean
+clean:
+	go clean 
+
+.PHONY: install
+install:
+	go install ./... 
+
+.PHONY: lint
+lint:
+	go get -v github.com/golang/lint/golint
```

Comment:
Updated snipped with `GOLINT ?= $(GOPATH)/bin/golint` to make GOLINT target more flexible